charts/trino/values.yaml

```
@@ -727,6 +727,54 @@ kafka:
   #    }
   # ```
 
+jmx:
+  # https://trino.io/docs/current/admin/jmx.html
+  enabled: false
+  registryPort: 9080
+  serverPort: 9081
+
+jmxExporter:
+  enabled: false
+  image: bitnami/jmx-exporter:latest
+  pullPolicy: Always
+  port: 5556
+  configProperties: []
+  # configProperties -- JMX Config Properties is mounted to /etc/jmx-exporter/jmx-exporter-config.yaml
```

You have to use the fully qualified property name in the beginning of the comment.
```suggestion
  # jmxExporter.configProperties -- JMX Config Properties is mounted to /etc/jmx-exporter/jmx-exporter-config.yaml
```